src/Repository/SessionTypeRepository.php

```diff
@@ -283,4 +284,39 @@ protected function attachCriteriaToQueryBuilder(QueryBuilder $qb, $criteria, $or
 
         return $qb;
     }
+
+    public function import(array $data, string $type, array $referenceMap): array
+    {
+        return match ($type) {
+            'session_type' => $this->importSessionTypes($data, $type, $referenceMap),
+            'session_type_x_aamc_method' => $this->importSessionTypesMethodMapping($data, $referenceMap),
+        };
+    }
+
+    protected function importSessionTypes(array $data, $type, array $referenceMap): array
```

Comment:
A bit picky, but this now imports a single session type and not multiple types so should be `importSessionType`

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: :heavy_check_mark: